app/scripts/views/sign_up.js

```diff
@@ -90,6 +91,14 @@ function (Cocktail, _, p, BaseView, FormView, Template, Session, AuthErrors,
                   AuthErrors.toError('SIGNUP_EMAIL_BOUNCE'));
       }
 
+      if (this.automatedBrowser) {
```

Comment:
This affects PR #2125 where `automatedBrowser` is no longer defined on the view.